### PR TITLE
:art: remove extra spaces, add exception

### PR DESCRIPTION
**Pull request by ShuffleZZZ**

### CODE REVIEW

**Comment by nothingelsematters** [comment]
I'll consider adding `ktlint` integration to fuck shitcode of ppl trying to merge their changes to develop asap.

File: src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -2,16 +2,19 @@ import com.elbekD.bot.Bot
...
-    val bot = Bot.createWebhook(userName,token) {
-        url = "${app}/${token}"
+    val bot = Bot.createWebhook(USER_NAME,token) {
+        url = "${APP_URL}/${token}"
         allowedUpdates = listOf(AllowedUpdate.Message)
+
         server {
             host = "0.0.0.0"
-            port = System.getenv("PORT").toInt()
+            port = (System.getenv("PORT") ?: DEFAULT_PORT).toInt()
```
Comment (nothingelsematters) [inline src/main/kotlin/Main.kt:17]: Maybe `const val DEFAULT_PORT = 80` and `port = System.getenv("PORT")?.toInt() ?: DEFAULT_PORT`?

Comment (ShuffleZZZ) [reply src/main/kotlin/Main.kt:17]: > Maybe `const val DEFAULT_PORT = 80` and `port = System.getenv("PORT")?.toInt() ?: DEFAULT_PORT`?

Elvis operator would work the same way after `toInt()`

Comment (nothingelsematters) [reply src/main/kotlin/Main.kt:17]: What's the difference then?

Comment (nothingelsematters) [reply src/main/kotlin/Main.kt:17]: > function of convertion would crush on null

It won't due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Comment (ShuffleZZZ) [reply src/main/kotlin/Main.kt:17]: > What's the difference then?

Let's get to type-level:

`System.getenv` returns `String?`

`toInt()` gets `String`

See the issue?

Comment (nothingelsematters) [reply src/main/kotlin/Main.kt:17]: > It won't crush due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Safe access operator again.

`System.getenv("PORT") -> null / PORT`

Comment (ShuffleZZZ) [reply src/main/kotlin/Main.kt:17]: > > function of convertion would crush on null
> 
> It won't crush due to safe access operator `?.`: `System.getenv("PORT")?.toInt()`.

Oh, didn't see what you did there

**Pull request merged**